q_yaar_platform/common/abstract_models.py

```diff
@@ -0,0 +1,51 @@
+"""
+Base behaviours. Can be inherited by other models.
+"""
+
+from uuid import UUID
+
+from django.db import models
+
+from common.uuid import unique_uuid4
+
+
+class AbstractExternalFacing(models.Model):
```

Comment:
Abstract models that can be inherited by any class (models.py in other modules).